tests/test_backend.py

```diff
@@ -1957,6 +1957,76 @@ def test_reverse_sequence_time_major(self):
         _ = tf.identity(x_, name=_TFOUTPUT)
         self._run_test_case([_OUTPUT], {_INPUT: x_val})
 
+    @skip_opset(9, "ReverseV2")
+    def test_reverse_v2(self):
+        # Tests for constant axis.
+        x_val = np.array([[[[0, 1, 2, 3], [4, 5, 6, 7], [8, 9, 10, 11]],
+                            [[12, 13, 14, 15], [16, 17, 18, 19], [20, 21, 22, 23]]]],
+                            dtype=np.float32)
+        x = tf.placeholder(tf.float32, [1, 2, 3, 4], name=_TFINPUT)
+        x_ = tf.reverse_v2(x, axis=[3])
+        _ = tf.identity(x_, name=_TFOUTPUT)
+        self._run_test_case([_OUTPUT], {_INPUT: x_val})
+        tf.reset_default_graph()
+
+        # Empty axis vector.
+        x_val = np.array([[[0, 1, 2, 3], [4, 5, 6, 7], [8, 9, 10, 11]],
```

Comment:
not need to assign a same value again, since it's same as line 1963

Reply:
could you split these tests into several test functions? otherwise it's hard for us to debug if something went wrong.

Reply:
Alright.

Reply:
and you could use np.random to generate input data, not need to specify it.